tests/run/dict_unpacking_not_for_arg_T4082.pyx

```diff
@@ -0,0 +1,21 @@
+cimport cython
+
+
+@cython.test_assert_path_exists('//MergedDictNode')
```

Comment:
I don't think we need `test_assert_path_exists` - I think the important test how it acts rather than how Cython stores it internally.